std/algorithm/setops.d

```diff
@@ -1019,6 +1117,12 @@ Lazily computes the intersection of two or more input ranges $(D
 ranges). The ranges are assumed to be sorted by $(D less). The element
 types of the ranges must have a common type.
 
+If all input ranges are sets (i.e. without duplicated elements),
+the resulting range is going to be a set, otherwise the result
+is going to be a multiset. In the case of multisets, the range with
```

Comment:
No, as shown above intersecting multisets may produce a set. Just delete the first sentence in this paragraph; the second contains all needed information.